Tasks/GoTool/gotool.ts

```diff
@@ -62,28 +55,24 @@ async function acquireGo(version: string): Promise<string> {
     try {
         downloadPath = await toolLib.downloadTool(downloadUrl);
     } catch (error) {
+        tl.debug(error);
         throw (tl.loc("VersionNotSupported", version));
     }
 
     //
     // Extract
     //
     let extPath: string;
-    try {
-        if (osPlat == 'win32') {
-            tl.assertAgent('2.115.0');
```

Comment:
should we keep the assert statement?

Reply:
Done

Reply:
Done
changes incorporated in https://github.com/Microsoft/vsts-tasks/pull/6307